vispy/visuals/volume.py

```diff
@@ -398,7 +446,9 @@
                 loc += small_step;
             }
             frag_depth_point = max_loc_tex * u_shape;
-            gl_FragColor = applyColormap(maxval);
+            vec4 max_color = $get_data(max_loc_tex);
```

Comment:
why is this not the same as `maxval`?

Reply:
The difference is `max_color` is a `vec4`, but `val` is `float`. I think I can refactor this because I don't like this extra texture lookup.